browser/ui/views/brave_javascript_tab_modal_dialog_view_views.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_JAVASCRIPT_TAB_MODAL_DIALOG_VIEW_VIEWS_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_JAVASCRIPT_TAB_MODAL_DIALOG_VIEW_VIEWS_H_
+
+#include "base/memory/weak_ptr.h"
+#include "chrome/browser/ui/views/javascript_tab_modal_dialog_view_views.h"
+#include "ui/base/metadata/metadata_header_macros.h"
+#include "ui/views/widget/widget_observer.h"
+
+namespace web_modal {
+class WebContentsModalDialogHost;
+}  // namespace web_modal
+
+class BraveJavaScriptTabModalDialogViewViews
```

Comment:
It looks a bit odd to me that we would use `ViewViews`. Granted I'm not that familiar with UI terminology, but I'm struggling to see what this repetition is supposed to say.